src/app/Shared/Services/Api.service.tsx

```diff
@@ -66,11 +66,10 @@ import {
 import { isHttpError, includesTarget, isHttpOk, isXMLHttpError } from './api.utils';
 import { LoginService } from './Login.service';
 import { NotificationService } from './Notifications.service';
-import { SessionState } from './service.types';
 import { TargetService } from './Target.service';
 
 export class ApiService {
-  private readonly archiveEnabled = new ReplaySubject<boolean>(1);
+  private readonly archiveEnabled = new ReplaySubject<boolean>(true);
```

Comment:
I think this should be `BehaviorSubject`.

![Screenshot_2024-06-11_14-49-10](https://github.com/cryostatio/cryostat-web/assets/3787464/7aea2f31-4dd5-4199-80c1-eb36e1ac9ad5)